
From: Andi Kleen <ak@suse.de>

Prevents some noise during boot up when no MD volumes are found.

I think I picked it up from someone else, but I cannot remember from whom
(sorry)

Cc: Neil Brown <neilb@cse.unsw.edu.au>

Signed-off-by: Andrew Morton <akpm@osdl.org>
---

 25-akpm/drivers/md/md.c |    2 +-
 1 files changed, 1 insertion(+), 1 deletion(-)

diff -puN drivers/md/md.c~make-md-no-device-warning-kern_warning drivers/md/md.c
--- 25/drivers/md/md.c~make-md-no-device-warning-kern_warning	Thu Aug  5 14:34:50 2004
+++ 25-akpm/drivers/md/md.c	Thu Aug  5 14:34:50 2004
@@ -406,7 +406,7 @@ static int read_disk_sb(mdk_rdev_t * rde
 	return 0;
 
 fail:
-	printk(KERN_ERR "md: disabled device %s, could not read superblock.\n",
+	printk(KERN_WARNING "md: disabled device %s, could not read superblock.\n",
 		bdevname(rdev->bdev,b));
 	return -EINVAL;
 }
_
